<launch>
    <arg name="veh"/>
    <arg name="pkg_name" value="led_interpreter"/>
    <arg name="node_name" default="led_interpreter_node"/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>

    <arg name="protocol_file_name" default="LED_protocol"/>
    <arg name="protocol_node_name" default="led_emitter_node"/>
    <arg name="intersectionType" />

    <node ns="$(arg veh)"   pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" required="true">
        <rosparam command="load" file="$(find led_emitter)/config/$(arg protocol_node_name)/$(arg protocol_file_name).yaml"/>
        <rosparam command="load" file="$(find led_interpreter)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
        <param name="intersectionType" value="$(arg intersectionType)"/>
    </node>
</launch>
